specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-11-01/providers.json

```diff
@@ -0,0 +1,288 @@
+          "description": "Name of metric specification."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Display name of metric specification."
+        },
+        "displayDescription": {
+          "type": "string",
+          "description": "Display description of metric specification."
+        },
+        "unit": {
+          "type": "string",
+          "description": "The metric unit. Possible values include: 'Bytes', 'Count', 'Milliseconds'."
+        },
+        "aggregationType": {
+          "type": "string",
+          "description": "The metric aggregation type. Possible values include: 'Average', 'Count', 'Total'."
+        },
+        "supportedAggregationTypes": {
+          "type": "array",
+          "description": "The supported aggregation types for the metrics.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "supportedTimeGrainTypes": {
+          "type": "array",
+          "description": "The supported time grain types for the metrics.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "lockAggregationType": {
+          "type": "string",
+          "description": "The metric lock aggregation type."
+        },
+        "dimensions": {
+          "type": "array",
+          "description": "The dimensions of metric",
+          "items": {
+            "$ref": "#/definitions/DimensionProperties"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "fillGapWithZero": {
+          "type": "boolean",
+          "description": "Property to specify whether to fill gap with zero."
+        },
+        "internalMetricName": {
+          "type": "string",
+          "description": "The internal metric name."
+        }
+      },
+      "type": "object"
+    },
+    "DimensionProperties": {
+      "description": "Type of operation: get, read, delete, etc.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of dimension."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Display name of dimension."
+        },
+        "toBeExportedForShoebox": {
+          "type": "boolean",
+          "description": "Property to specify whether the dimension should be exported for Shoebox."
+        }
+      },
+      "type": "object"
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Client Api Version."
+    }
+  },
```

Comment:
You are highly recommended to remove your own definitions of these and use the definitions from common-specification/types/v3/types.json instead

Reply:
This is old existing file, not introduced in this PR, so skipping.